pkgs/servers/aws-codeartifact-proxy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Comment:
New packages go in pkgs/by-name nowadays

Reply:
Yep! I moved the file to comply with that convention just now in b4ea51d6d2fb28ea7bb20dbbe5705d853ad0c6a2.